index.src.html

```diff
@@ -561,7 +566,7 @@ <h4 id="send-reports" algorithm>Send reports</h4>
 
           3.  If |result| is "`Remove Endpoint`":
 
-              1.  Remove |endpoint| from |document|'s {{document/endpoints}}
+              1.  Remove |endpoint| from |context|'s {{document/endpoints}}
                   list.
```

Comment:
Similarly, any reference to `{{document/endpoints}}` seems like it needs updating.

Reply:
I searched for "endpoints<", but missed these :( Thanks for catching.